go-update/src/main/java/org/reactome/release/goupdate/GoTermInstanceModifier.java

```diff
@@ -499,5 +561,14 @@ public void updateRelationship(Map<String, List<GKInstance>> allGoInstances, Map
 			}
 		}
 	}
-
+	
+	private static String abbreviate(String s, int maxLength)
+	{
+		return s.substring(0,Math.min(s.length(), maxLength)) + ( s.length() > maxLength ? "..." : "" );
+	}
+	
+	private static String abbreviate(String s)
+	{
+		return abbreviate(s, 47);
```

Comment:
Why 47?  This looks like a magic number - could it be given a named constant?

Reply:
Originally, I wanted to limit to 50 characters, for readability. Because there is a lot of "header" such as "[GO_BiologicalProcess:1234] The blah blah blah..." it seemed that 50 characters gave good results. 47 is to make sure that _exactly_ 50 characters would be used because the last three characters would be: "..." ;)